test/run_cargo_test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+test_options=
+cargo_option=
+component=
+while [ "$1" != "" ]; do
+  case $1 in
+    -c | --component )      shift
+                            component=$1
+                            ;;
+    -o | --cargo-options )  shift
```

Comment:
Since the only thing `--cargo-options` is used for is `--features` why not change this to `--features`, giving nicer ergonomics:
```bash
./test/run_cargo_test.sh --component sup --cargo-options "--features inconsistent_tests integration_tests"
```
becomes (with my other suggestions)
```bash
./test/run_cargo_test.sh --features inconsistent_tests integration_tests sup
```